Makefile

```diff
@@ -7,7 +7,16 @@ build: recipe.yaml
 	ansible-bender build --build-volumes $(CURDIR):/src:Z -- ./recipe.yaml $(BASE_IMAGE) $(OGR_IMAGE)
 
 prepare-check:
-	sudo dnf install python3-tox python36
+	if ! test -x /usr/bin/python3.6 && \
+	   ! test -x /usr/bin/python3.7 && \
+	   ! test -x /usr/bin/python3.8 && \
+	   ! test -x /usr/bin/python3.9 ; \
+	then \
+		sudo dnf -y install python36 ; \
+	fi
```

Comment:
I am not sure if we need to take care of Python installation, what others think?

* For each new version of Python, we will need to update this. 
* How possible is not having Python installed? CentOS containers?

Reply:
I agree, I don't think it's necessary to check for the Python version;
installation of latest Python3 should be taken care of by installation
of python3-flexmock, python3-requre.

I think that absence of CentOS containers may be taken care of by
checking for installation of ansible-bender and podman; see below.